tools/integration-tests/runTests.sh

```diff
@@ -2,8 +2,8 @@
 
 TEST_NAMES='autopeering common drng message value consensus faucet syncbeacon mana'
 
-DOCKER_BUILDKIT=1
-COMPOSE_DOCKER_CLI_BUILD=1
+export DOCKER_BUILDKIT=1
+export COMPOSE_DOCKER_CLI_BUILD=1
```

Comment:
Oh shame on me! Thanks for fixing this. 
Please do the same in this file as well: `tools/docker-network/run.sh`